ipv8/attestation/identity/community.py

```diff
@@ -31,7 +33,7 @@ def add_known_hash(self, attribute_hash, name, public_key, metadata=None):
     def get_attestation_by_hash(self, attribute_hash):
         blocks = self.persistence.get_all_blocks()
         for block in blocks:
-            if block.transaction and block.transaction.get("hash", None) == attribute_hash:
+            if block.transaction and block.transaction.get(b"hash", None) == attribute_hash:
```

Comment:
Could you elaborate on why keys in a TrustChain transaction became `byte`? I would argue these strings are (always?) human-readable so there is no need to make them byte.